app/src/main/java/com/teamltt/carcare/database/contract/UserContract.java

```diff
@@ -1,41 +1,65 @@
 /*
- * Copyright 2017, Team LTT
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ ** Copyright 2017, Team LTT
+ **
+ ** Licensed under the Apache License, Version 2.0 (the "License");
+ ** you may not use this file except in compliance with the License.
+ ** You may obtain a copy of the License at
+ **
+ **     http://www.apache.org/licenses/LICENSE-2.0
+ **
+ ** Unless required by applicable law or agreed to in writing, software
+ ** distributed under the License is distributed on an "AS IS" BASIS,
+ ** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ** See the License for the specific language governing permissions and
+ ** limitations under the License.
  */
 
 package com.teamltt.carcare.database.contract;
 
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
 public class UserContract {
 
     public static final String SQL_CREATE_ENTRIES = "CREATE TABLE " + UserEntry.TABLE_NAME + " (" +
             // user_id INTEGER PRIMARY KEY
             UserEntry.COLUMN_NAME_ID + " INTEGER PRIMARY KEY AUTOINCREMENT," +
-            UserEntry.COLUMN_NAME_FB_API_KEY + " TEXT," +
+            UserEntry.COLUMN_NAME_GOOGLE_USER_ID + " TEXT," +
             UserEntry.COLUMN_NAME_FIRST_NAME + " TEXT," +
             UserEntry.COLUMN_NAME_LAST_NAME + " TEXT" +
             ");";
 
     public static final String SQL_DROP_ENTRIES = "DROP TABLE IF EXISTS " + UserEntry.TABLE_NAME;
 
+    public static Cursor queryUserID(SQLiteDatabase db, String google_user_id) {
+        String table = UserEntry.TABLE_NAME;
+        String[] columns = {
+                UserEntry.COLUMN_NAME_ID,
+                UserEntry.COLUMN_NAME_GOOGLE_USER_ID,
+                UserEntry.COLUMN_NAME_FIRST_NAME,
+                UserEntry.COLUMN_NAME_LAST_NAME
+        };
+        String where_cond = "UserEntry.COLUMN_GOOGLE_USER_ID = " + google_user_id;
```

Comment:
Don't hardcode the key, instead change this to `String where_cond = UserEntry.COLUMN_NAME_GOOGLE_USER_ID + " = ?";`, then make a `String[] args = {google_user_id};`, and pass it in as an argument after the `where_cond`